idl/github.com/uber/cadence/cadence.thrift

```diff
@@ -19,6 +19,7 @@
 // THE SOFTWARE.
 
 include "shared.thrift"
+include "replicator.thrift"
```

Comment:
You can move the def from replicator to shared without breaking anything.

Reply:
I cannot unless I move the ReplicationTask definition to shared as well. Then it means that we move all the stuffs in repliactor to shared.

Reply:
i see